practice-one/src/layouts/LeftSidebar/LeftSidebar.jsx

```diff
@@ -6,6 +6,9 @@ import logoBrand from "../../assets/images/wons-logo.svg";
 import lantern from "../../assets/images/lantern.svg";
 import lightLantern from "../../assets/images/light-lantern.svg";
 import { TAG_LINKS } from "../../data/tagLinks";
+import Button from "../../components/Button/Button";
+import { BUTTON_VARIANTS } from "../../constants/buttons";
+import Account from "../../components/Account/Account";
```

Comment:
group import the same scope

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/8a231b54-5e30-4c49-b89a-f474f2ddb632)